src/sympleq/core/paulis/pauli_sum.py

```
@@ -769,26 +769,39 @@ def combine_equivalent_paulis(self):
         """
         Combines equivalent Pauli operators in the sum by summing their coefficients and deleting duplicates.
         """
-        # self.standardise()  # makes sure all phases are 0
-        # combine equivalent Paulis
-        to_delete = []
-        for i in reversed(range(self.n_paulis())):
-            ps1 = self.select_pauli_string(i)
-            for j in range(i + 1, self.n_paulis()):
-                ps2 = self.select_pauli_string(j)
-                if ps1 == ps2:
-                    # FIXME: can overflow for very large n_paulis.
-                    #        One solution could be to normalize it by dividing by the smallest weight.
-                    self._weights[i] = self.weights[i] + self.weights[j]
-                    to_delete.append(j)
-        self._delete_paulis(to_delete)
+        if self.n_paulis() == 0:
+            return
```

`return` what? Not following here but most likely because I am missing something... Maybe raise an error instead of return?